tests/unit/frameworks/pytorch/restore_utils_test.py

```diff
@@ -139,20 +139,22 @@ def test_load_full(self):
             tot_eq = (param1 == param2).sum().item()
             self.assertNotEqual(tot_eq, np.prod(param1.shape))
 
-        # Check output through the full network.
-        out = full_forward(model, self.in_1)
-        num_matches = out.isclose(self.out_full, atol=1e-2).sum().item()
-        self.assertEqual(num_matches, 1)  # some correct
-
-        # Check output through the lower network.
-        out = lower_forward(model, self.in_1)
-        num_matches = out.isclose(self.out_lower, atol=1e-2).sum().item()
-        self.assertEqual(num_matches, 1337)  # some correct
-
-        # Check output through the lower network.
-        out = upper_forward(model, self.in_2)
-        num_matches = out.isclose(self.out_upper, atol=1e-2).sum().item()
-        self.assertEqual(num_matches, 1)  # some correct
+        # These exact numbers are random.
```

Comment:
Instead of commenting out the test could you add a "skip" comment just like you added to `test_checkpoint_backward_compatibility`?

Reply:
But some parts of these tests are still valid.

Reply:
In that case we can either break these testes into 2 different parts or remove the commented out code if is no longer relevant. 
It is just hard to maintain commented out code.